docs/registry/_index.md

```diff
@@ -40,6 +41,22 @@ See [Machine Learning](/ml/) for more information.
 The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
 You can also [upload your own model to the registry](/ml/upload-model/).
 
+You can search the available ML models from the Viam registry here:
+
+<div id="searchbox"></div>
+<p>
+<div id="searchstats"></div></p>
+<div class="mr-model" id="">
+  <div class="modellistheader">
+    <div class="type">API</div>
+    <div class="name">Model</div>
+    <div>Description</div>
+  </div>
+<div id="hits" class="modellist">
+</div>
+<div id="pagination"></div>
+</div>
```

Comment:
id's should be unique across a page. In this case this is part of why this currently isn't working. If you read through the bottom half of the mlmodels.js script it does reference #hits and #searchbox and #searchstats and #pagination. That is why it's not working. It's all name collisions here as well

Reply:
ahh!! Thanks for explaining!